packages/forms/frontend/sirius-components-forms/src/propertysections/__tests__/__snapshots__/ListPropertySection.test.tsx.snap

```diff
@@ -905,3 +905,153 @@ exports[`should the click event sent on item click 2`] = `
   </div>
 </div>
 `;
+
+exports[`should the click event sent on item click 3`] = `
```

Comment:
Not sure where this comes from. Running `vitest` with it present tells me "Snapshots  1 failed" (but the tests are green anyway) and `vitest -u` removes it (which makes the failure message go away).

Reply:
Done.